included a custom toggle event

marcos-tulio opened the pull request:
I included a custom toggle event with JavaScript only (without JQuery). And an example page.

Commit:
included custom ontoggle event

Commit:
included a small example

Commit:
added an example page to the ontoggle event

marcos-tulio committed:
included a small example

gitbrent commented:
@mrcapybara 

What's the difference between this and `change`?

bootstrap-switch-button/demo/index.html#events

```javascript
$('#switch-event').change(function() {
    //alert("Button state: " + swtButton.checked);
    alter('Switch-Button checked? = ' + $(this).prop('checked'));
})
```

marcos-tulio commented:
@gitbrent 

In this case, it is the same. 

I fixed the issue #6 with this, in my case (I used customs bootstrap files and didn't use JQuery). I implemented the `ontoggle` event so as not to override your `change` implementations. I tested your update and all it's working fine.

My alternative fix it's not recommended. Please, close and discard this request.

Ps.: This reference is broken:

` <script src="../js/bootstrap-switch-button.js"></script>`

in  `bootstrap-switch-button/test/test-all-sizes.html` line 19.

gitbrent commented:
@mrcapybara - thanks!
